Lib/profiling/sampling/live_collector/collector.py

```diff
@@ -896,19 +928,148 @@ def _handle_input(self):
             if self._trend_tracker is not None:
                 self._trend_tracker.toggle()
 
-        elif ch == curses.KEY_LEFT or ch == curses.KEY_UP:
-            # Navigate to previous thread in PER_THREAD mode, or switch from ALL to PER_THREAD
+        elif ch == ord("j") or ch == ord("J"):
```

Comment:
Unless I'm missing something, I think we can deduplicate a lot of logic with some helpers for up/down movement, right?